core.Dockerfile

```diff
@@ -1,7 +1,7 @@
 ###############################################################################
 # Builder image
 ###############################################################################
-FROM redhat/ubi9 AS builder
+FROM --platform=$BUILDPLATFORM redhat/ubi9 AS builder
```

Comment:
Who should set those variables?  Explicitly in our Makefile ?

Reply:
The docker doc says they are set when we use BuildKit:
https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope

So when we use `buildx`, values about the build platform are automatically available, values about the target platform (Yes, you are right) are passed in by the `--platform` flag of `buildx build` which is in our Makefile.